### ISSUE TITLE
Support for HDP 2.5

### ISSUE THREAD

**Issue by jvalkeal** [spring-cloud/spring-cloud-dataflow-ambari-rpm#20]
Need to modify rpm scripts so that 2.5 gets integrated correctly.

**Comment by jvalkeal** [comment]
Some python functions has been changed:

```
stderr: 
Traceback (most recent call last):
  File "/var/lib/ambari-agent/cache/stacks/HDP/2.2/services/SCDF/package/scripts/scdf_shell.py", line 39, in <module>
    ScdfShell().execute()
  File "/usr/lib/python2.6/site-packages/resource_management/libraries/script/script.py", line 280, in execute
    method(env)
  File "/var/lib/ambari-agent/cache/stacks/HDP/2.2/services/SCDF/package/scripts/scdf_shell.py", line 31, in install
    self.configure(env)
  File "/var/lib/ambari-agent/cache/stacks/HDP/2.2/services/SCDF/package/scripts/scdf_shell.py", line 36, in configure
    scdf(name='shell')
  File "/var/lib/ambari-agent/cache/stacks/HDP/2.2/services/SCDF/package/scripts/scdf.py", line 48, in scdf
    recursive=True
  File "/usr/lib/python2.6/site-packages/resource_management/core/base.py", line 147, in __init__
    raise Fail("%s received unsupported argument %s" % (self, key))
resource_management.core.exceptions.Fail: Directory['/var/log/scdf'] received unsupported argument recursive
```

Looks like parameter name has been changed.

```
   Directory(params.limits_conf_dir,
-            recursive=True,
+            create_parents = True,
```

**Comment by jvalkeal** [comment]
Need to investigate how new version definition file works in 2.5 as for default choice new yum repos are not added but those are available if you choose 2.5(default version definition) during a cluster install.

**Comment by jvalkeal** [comment]
Keeping this as is. Possible work for extension/mpack in a separate tickets in a future.
